ddr/ddr/GameScene.swift

```diff
@@ -31,12 +76,12 @@ class GameScene: SKScene, SKPhysicsContactDelegate {
   
   var velocity : CGFloat = 0.02
   
-  let audioManager = AudioManager()
+  
   let motionManager = CMMotionManager()
```

Comment:
We can probably make this a singleton along with audio manager

Reply:
Yes, this is probably the best thing to do. I am not super familiar with the pattern so I have not done so.

I have created an issue #65 for this task.